glean-core/src/upload/request.rs

```diff
@@ -88,11 +88,25 @@ impl PingRequest {
         gzipper.finish().ok()
     }
 
+    /// Creates a formatted date string that can be used with Date headers.
+    fn create_date_header_value(current_time: DateTime<Utc>) -> String {
```

Comment:
Not wrong or anything, but this could just be a free-standing function in this module.

Reply:
Next time @badboy , next time! :-)